tests/env_helpers.go

```diff
@@ -0,0 +1,205 @@
+package tests
```

Comment:
Mind moving this under internal to avoid making this part of the public API?

Reply:
Is this still needed now that `tests` is a separate submodule?